src/test/crimson/cbt/radosbench_4K_read.yaml

```diff
@@ -18,7 +18,7 @@ tasks:
         acceptable:
           bandwidth: '(or (greater) (near 0.05))'
           iops_avg: '(or (greater) (near 0.05))'
-          iops_stddev: '(or (less) (near 0.20))'
+          iops_stddev: '(or (less) (near 0.50))'
```

Comment:
Maybe we should consider dropping `iops_stddev` entirely.

Reply:
Do you mean just the requirement to pass, or stop recording it?  I think we should still record it, and perhaps in the future we'll be able to narrow it down.